client/src/main/java/io/pravega/client/stream/impl/CheckpointState.java

```diff
@@ -259,7 +268,28 @@ private void write00(CheckpointState object, RevisionDataOutput output) throws I
             output.writeCollection(object.checkpoints, stringSerializer);
             output.writeMap(object.uncheckpointedHosts, stringSerializer, (out, hosts) -> out.writeCollection(hosts, stringSerializer));
             output.writeMap(object.checkpointPositions, stringSerializer, (out, map) -> out.writeMap(map, segmentSerializer, longSerializer));
+        }
+
+        private void read01(RevisionDataInput input, CheckpointStateBuilder builder) throws IOException {
```

Comment:
Same situation here.

Reply:
This was existing code and ideally I should not be changing it as part of this PR. But will fix.

Reply:
fixed